lib/livebook/notebook/explore/intro_to_maplibre.livemd

```diff
@@ -77,22 +77,16 @@ Calling the `Ml.new/0` function with no arguments will render a simple map with
 Ml.new()
 ```
 
-You can easily override this initial style by declaring your own style using the `:style` option as well as any other root-level properties you wish to configure. It can be either a URL, a JSON string or a map.
+You can easily override this initial style by declaring your own style using the `:style` option as well as any other root-level properties you wish to configure. It can be either a URL (requires the [`req`](https://hex.pm/packages/req) package), a `map` or one of the built-in styles we offer, `:street` and `:terrain`.
```

Comment:
```suggestion
You can easily override this initial style by declaring your own style using the `:style` option as well as any other root-level properties you wish to configure. It can be either a URL (requires the [`req`](https://hex.pm/packages/req) package), a `map` or one of the built-in styles, such as `:street` or `:terrain`.
```